packages/tokens/README.md

```diff
@@ -0,0 +1,38 @@
+# @vapor/tokens
+
+Design tokens of the Vapor Design System extracted from Figma.
+
+#### What is a design token?
+
+Simply put, a design token is a stored design decision. It can be understood as a variable that holds the result of a choice. Decisions can be made about multiple different subjects, for example colors, typography, spacing, etc. Therefore, it is important to consider that design tokens come in as many types as there are design subjects.
```

Comment:
```suggestion
Simply put, a design token is a stored design decision. It can be understood as a variable that holds the result of a choice (in this case made in Figma). Decisions can be made about multiple different subjects, for example colors, typography, spacing, etc. Therefore, it is important to consider that design tokens come in as many types as there are design subjects.
```